<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BBoxViewer</class>
 <widget class="QWidget" name="BBoxViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>858</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CaseBBoxes viewer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lineEditPath">
       <property name="placeholderText">
        <string>Select label imageâ€¦</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnBrowse">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnLoad">
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="topMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="fileName">
       <property name="text">
        <string>FileName</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labels">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="caseID">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="topMargin">
      <number>20</number>
     </property>
     <item>
      <widget class="QTreeView" name="treeBBoxes">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="VTKLabelmapView" name="viewPort" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>500</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VTKLabelmapView</class>
   <extends>QWidget</extends>
   <header>vtklabelmapview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
